src/OpenIddict.Core/OpenIddictManager.cs

```diff
@@ -0,0 +1,78 @@
+﻿using System;
+using System.Threading.Tasks;
+using Microsoft.AspNet.Http;
+using Microsoft.AspNet.Identity;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Internal;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.OptionsModel;
+
+namespace OpenIddict {
+    public class OpenIddictManager<TUser, TApplication> : UserManager<TUser> where TUser : class where TApplication : class {
```

Comment:
IMHO, inheritance is quite handy, as you don't have to import 2 (or more) managers in your code to play with extremely-related stuff.